npm/private/npm_package_store.bzl

```diff
@@ -402,6 +402,12 @@ deps of npm_package_store must be in the same package.""" % (ctx.label.package,
         for npm_package_store_info in npm_package_store_infos:
             transitive_files_depsets.append(npm_package_store_info.files)
 
+    for jsinfo in js_infos:
+        for source in jsinfo.sources.to_list():
```

Comment:
We can't use `to_list()` like this and will need a better solution if a fix like this is required